docs/xml-config-file.rst

```diff
@@ -142,12 +147,15 @@ Each property set will be sent to its component as a vector of pairs of strings.
 Connectors
 ----------
 
-Any number of connectors can be defined in our XML configuration file, but at least one is needed to make IS perform any work.
-They must contain a subscriber and a publisher. Each of them is configured by a participant or bridge name and the subscriber's or publisher's name respectively.
+The *connectors* are just relationships between subscribers and publishers, and optionally, a transformation function.
+Any number of connectors can be defined in our XML configuration file,
+but at least one is needed to make IS perform any work.
+They must contain a subscriber and a publisher.
+Each of them is configured by a participant or bridge name and the subscriber's or publisher's name respectively.
 
-In the follow example, we define a connector whose subscriber receives data from Fast-RTPS, and its publisher
-writes that data to a text file. A :ref:`transformation libraries`'s function that adds the timestamp before the data is wrote is defined
-too.
+In the following example, we define a connector whose subscriber receives data from Fast-RTPS, and its publisher
+writes that data to a text file.
+A :ref:`transformation libraries`'s function that adds the timestamp before the data is wrote is defined too.
```

Comment:
xml is wrong. in subscriber the attribute should be `participant_profile` and `subscriber_profile`.

Reply:
Fixed.